### DISCUSSION TITLE
Split the window by default when multiple files are speficied

### DISCUSSION THREAD

**Discussion by dsseng** [helix-editor/helix#10602] [General]
#### Describe your feature request
When Helix is started with a command like `hx README.md CONTRIBUTING.md` it would be great to split window into half, then half and 2 quarters, 4 quarters etc. It should be expected to have views side-by-side, but an option still will be better to suit everyone's needs.

**Comment by archseer** [comment]
I think this was the behavior at some point but it was changed if you're opening a bunch of files that would result in too many splits

**Comment by dsseng** [comment]
Well, it should be restored but limited. Probably based on current console resolution.

**Comment by EpocSquadron** [comment]
How about a flag specifying the number of splits one wants?

**Comment by dsseng** [comment]
I'd prefer a config option for that. As I think it would work: first n items get distributed using split layout, others are left loaded but not displayed.
